BREAKING-CHANGES.md

```diff
@@ -385,13 +385,13 @@ The MSRV of ratatui is now 1.67 due to an MSRV update in a dependency (`time`).
 
 ## [v0.22.0](https://github.com/ratatui-org/ratatui/releases/tag/v0.22.0)
 
-### bitflags updated to 2.3 ([#205])
+### `bitflags` updated to 2.3 ([#205])
 
 [#205]: https://github.com/ratatui-org/ratatui/issues/205
 
-The serde representation of bitflags has changed. Any existing serialized types that have Borders or
-Modifiers will need to be re-serialized. This is documented in the [bitflags
-changelog](https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200-rc2)..
+The `serde` representation of `bitflags` has changed. Any existing serialized types that have Borders or
+Modifiers will need to be re-serialized. This is documented in the [`bitflags`
+change log](https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200-rc2)..
```

Comment:
Let's add this to the typos dictionary - it's reasonable to use this as a correct word.

Reply:
So… 'changelog' or 'change log'?

Reply:
"changelog"

Reply:
The tool `typos` is not annoyed by it. LanguageTool is. → No change in this repo required.

Reply:
```suggestion
changelog](https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200-rc2)..
```